src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingManagementController.java

```diff
@@ -141,7 +141,6 @@ public HearingResponse updateHearing(@RequestBody @Valid UpdateHearingRequest he
         @ApiResponse(code = 404, message = "001 No such id"),
```

Comment:
Not your changes, but these message values could reference the constants in `ValidationError`, rather than repeating the strings here.

`@ApiResponse(code = 404, message = ValidationError.HEARING_ACTUALS_INVALID_STATUS)`

Reply:
I made changing the messages into ValidationError a tech-debt ticket HMAN-166